services/ecs.py

```diff
@@ -2,52 +2,226 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("ecs")
 
 
 def ecs_awsvpc_networking_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    task_definitions = client.list_task_definitions(status="ACTIVE")["taskDefinitionArns"]
+    latest_task_definitions = {}
+
+    for task_definition in task_definitions:
+        if latest_task_definitions.get(task_definition.rsplit(":", 1)[0], 0) < int(task_definition.rsplit(":", 1)[1]):
+            latest_task_definitions[task_definition.rsplit(":", 1)[0]] = int(task_definition.rsplit(":", 1)[1])
+
+    for task_definition in latest_task_definitions.keys():
+        task_definition_arn = f"{task_definition}:{latest_task_definitions[task_definition]}"
+        task_definition = client.describe_task_definition(taskDefinition=task_definition_arn)["taskDefinition"]
```

Comment:
task definition arn을 처리하는 방식을 조금 더 간결하게 바꿀 수 있어 보입니다.
```suggestion
    for task_definition in task_definitions:
        family, revision = task_definition.rsplit(":", 1)        
        latest_task_definitions[family] = max(latest_task_definitions.get(family, 0), int(revision))

    for family, revision in latest_task_definitions.items():
        task_definition_arn = f"{family}:{revision}"
        task_definition = client.describe_task_definition(taskDefinition=task_definition_arn)["taskDefinition"]
```